src/endpoints/users-endpoint.ts

```diff
@@ -0,0 +1,26 @@
+import { Endpoint, Request, Response } from 'death-adder';
+
+class UsersEndpoint extends Endpoint {
+  // get all users
+  async get(request: Request, response: Response) {
+    const users = await request.attachments.db.query(
+      'SELECT username FROM users;'
+    );
+
+    response.json(users);
+  }
+
+  // update a user
+  async put(request: Request, response: Response) {
+    const { id, username, email, password } = request.body;
+    const update = await request.attachments.db.query(
+      `UPDATE users
+      SET username = ${username}, email = ${email}, password = ${password}
+      WHERE id=${id};`
```

Comment:
This is a direct SQL injection vulnerability, please use the query parameters instead of string interpolation. Imagine someone passes in an id of 
```SQL
(TRUNCATE comments RETURNING 1)
```